sdk/maps/azure-maps-timezone/src/main/java/com/azure/maps/timezone/TimezoneAsyncClient.java

```diff
@@ -244,8 +294,8 @@ public Mono<Response<List<TimezoneWindows>>> getWindowsTimezoneIdsWithResponse(J
      * @return this object is returned from a successful Timezone Enum Windows call.
      */
     @ServiceMethod(returns = ReturnType.SINGLE)
-    public Mono<List<TimezoneWindows>> getWindowsTimezoneIds(JsonFormat format) {
-        return this.serviceClient.getWindowsTimezoneIdsAsync(format);
+    public Mono<Response<List<TimezoneWindows>>> getWindowsTimezoneIdsWithResponse(Context context) {
```

Comment:
cannot be public in the AsyncClient. No AsyncClients can take a Context. This should be package visibility only, to help the Sync client.